installation/language/en-GB/joomla.ini

```
@@ -26,6 +26,22 @@ INSTL_WARNJSON="Your PHP installation needs to have JSON enabled for Joomla to b
 
 ; Precheck view
 INSTL_DATABASE_SUPPORT="Database Support:"
+INSTL_AUTOFIND_FTP_PATH="Autofind FTP Path"
+INSTL_FTP="FTP Configuration"
+INSTL_FTP_DESC="<p>On some servers you may need to provide FTP credentials for installation to complete. If you have difficulties completing installation without these credentials, check with your host to determine if this is necessary. </p><p>For security reasons, it is best to create a separate FTP user account with access to the Joomla! installation only and not the entire web server. Your host can help you with this.</p><p><b>Note:</b> If you are installing on a Windows Operating System, the FTP layer is <strong>not</strong> required.</p>"
+INSTL_FTP_ENABLE_LABEL="Enable FTP Layer"
+INSTL_FTP_HOST_LABEL="FTP Host"
+INSTL_FTP_PASSWORD_LABEL="FTP Password"
+INSTL_FTP_PORT_LABEL="FTP Port"
+INSTL_FTP_ROOT_LABEL="FTP Root Path"
+INSTL_FTP_SAVE_LABEL="Save FTP Password"
+INSTL_FTP_TITLE="FTP Configuration (<strong class=\"red\">Optional - Most Users Can Skip This Step - Press Next to Skip</strong>)"
+INSTL_FTP_USER_LABEL="FTP Username"
+INSTL_VERIFY_FTP_SETTINGS="Verify FTP Settings"
+INSTL_FTP_SETTINGS_CORRECT="Settings correct"
+INSTL_FTP_USER_DESC="Warning! It is recommended to leave this blank and enter your FTP username each time you transfer files."
+INSTL_FTP_PASSWORD_DESC="Warning! It is recommended to leave this blank and enter your FTP password each time you transfer files."
+INSTL_SKIP_FTP="Manually create file"
```

This is a button so it should be capitalised to follow the style guide

```suggestion
INSTL_SKIP_FTP="Manually Create File"
```